### ISSUE TITLE
per-class accuracy 

### ISSUE THREAD

**Issue by ghayth82** [rasbt/mlxtend#615]
hi
per-class accuracy is provide the average per-class accuracy , i suggest to add per-class accuracy . I mean to calculate accuracy of each class individually, so for example  to check which class affecting the model and has the lowest accuracy

**Comment by rasbt** [comment]
I think that might be useful in certain applications. It should probably be a standalone function `per_class_accuracy` (similar to `accuracy_score` in https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html) so that users can choose which class to use as the "positive class). I.e., adding it to the current `scoring` function wouldn't work, because it wouldn't allow that choice.

**Comment by rasbt** [comment]
Fixed in #624
